commerce-lang/src/main/resources/content/Language.properties

```diff
@@ -325,6 +325,8 @@ option-templates=Option Templates
 option-value-detail=Option Value Detail
 option-values=Option Values
 order-confirmation=Order Confirmation
+order-creator-definition-term=The user that created the order.
+order-id-definition-term=The order ID
```

Comment:
We should make these language keys consistent -- either they both end in a period, or neither does. I think since neither one is a complete sentence, it makes the most sense to leave them as simple phrases:

    The user that created the order
    The order ID